### PR TITLE
E2vid integration

### PR DESCRIPTION
**Pull request by friedhelm-h**
This implements a basic module E2Vid. It leaves it to the user to call it with subsequent event batches of a good size.

### CODE REVIEW

File: src/evlib/processing/reconstruction/e2vid/e2vid.py [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+import sys
+from typing import Tuple
+
+import numpy as np
+import urllib.request
+
+from .utils.loading_utils import load_model, get_device
+from .image_reconstructor import ImageReconstructor
+from .options.inference_options import set_inference_options
+from ....representation import VoxelGrid  # TODO: import correctly from evlib
```
Comment (friedhelm-h) [inline src/evlib/processing/reconstruction/e2vid/e2vid.py:13]: I was not sure here. Can also do 

`from evlib.representation import VoxelGrid`

?

File: src/evlib/processing/reconstruction/e2vid/e2vid.py [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+
+logging.basicConfig(level=logging.INFO, format='%(asctime)s [%(levelname)s] %(message)s')
+logger = logging.getLogger('ev_lib')
+
+MODEL_PATH = os.path.join(os.path.dirname(os.path.realpath(__file__)),
```
Comment (friedhelm-h) [inline src/evlib/processing/reconstruction/e2vid/e2vid.py:18]: I was not sure here. Is there a better way to define path to library root?

**Comment by shiba24** [comment]
Thanks for the PR, I will take a look. We will need to resolve a couple of things on this PR. Will let you know what/how.

**Comment by shiba24** [comment]
@friedhelm-h can you rebase the latest main?

File: src/evlib/processing/reconstruction/e2vid/image_reconstructor.py [hunk 3]
```diff
@@ -0,0 +1,102 @@
+from typing import Any, Union
+
+import torch
+import numpy as np
+
+from .model.model import *
+from .utils.inference_utils import CropParameters, EventPreprocessor, IntensityRescaler, ImageFilter, UnsharpMaskFilter
+from .utils.timers import CudaTimer
```
Comment (shiba24) [inline src/evlib/processing/reconstruction/e2vid/image_reconstructor.py:8]: Do you think we can entirely remove this CudaTimer?

File: src/evlib/processing/reconstruction/e2vid/utils/timers.py [hunk 4]
```diff
@@ -0,0 +1,57 @@
+import torch
```
Comment (shiba24) [inline src/evlib/processing/reconstruction/e2vid/utils/timers.py:1]: In general, I like to delete codes if possible. Can you delete this entire file?

**Review by shiba24 — COMMENTED** [review]
@friedhelm-h  Left a few comments. I will take a look further, and probably try to fix the test errors on Thu or Fri.

**Comment by shiba24** [comment]
@friedhelm-h merging - thanks for the work. let's move on, get things done!

**Pull request merged**